extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerConfig.java

```diff
@@ -118,4 +118,9 @@
     @ConfigItem(defaultValue = "true")
     public Boolean logTraceContext;
 
+    /**
+     * Whether the jaeger should run in zipkin compatibility mode
```

Comment:
Remove "the", as I don't think it's necessary for the sentence

Reply:
Removed